playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -192,7 +192,7 @@
     when: origin_reconcile_bindings | bool or ent_reconcile_bindings | bool
     register: reconcile_bindings_result
     changed_when:
-    - reconcile_bindings_result.stdout.length > 0
+    - reconcile_bindings_result.stdout != ''
```

Comment:
* There are many different ways to ~shave this yack~ write these `when` conditions
* I don't care which one we use but we need to make a decision
* I personally prefer the last one @detiber posted, it handles a lot of edge cases

```yaml
changed_when:
- reconcile_bindings_result.stdout | default(false, boolean=true)
```

If `stdout` is empty for some reason then the default is `false`. Otherwise the result is `true`.
* `stdout == ''` → **Nothing changed** → `default` filter says *no input value, so this is `false`*
* `stdout != ''` → **Something changed changed** → `default` filter says *input value present, so this is `true`*


@detiber @ewolinetz if that translation is correct ^, let's just do that and move on with this the way @detiber wrote it up.